### PR TITLE
[ES] Add `osu!catch` and `osu!taiko`

### PR DESCRIPTION
**Pull request by agatemosu**
Supersedes #9916 (applied my review + my review that was not sent)

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Game_mode/osu!taiko/es.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+Después de completar un beatmap, a la puntuación se le asigna a un [grado](/wiki/Gameplay/Grade#osu!taiko), una breve evaluación de la presición en forma de una sola letra. Una SS dorada o plateada denota una precisión del 100 %, y todo lo demás, de S a D, depende de la cantidad de 300, 100 y fallos.
+
+## Skinning
+
+*Consulta la [página de skinning de osu!taiko](/wiki/Skinning/osu!taiko) Para obtener información completa.*
+
+## Mapping en osu!taiko
+
+- Las notas rojas se refieren al círculo de golpe normal,
+  - las notas rojas grandes necesitan un hitound final.
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:101]:
```suggestion
  - las notas rojas grandes necesitan un hitsound final.
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+- Las notas azules deben tener un sonido de silbido/aplauso en ese círculo,
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:102]:
```suggestion
- Las notas azules deben tener un hitsound de silbido/aplauso en ese círculo,
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  - las notas azules grandes necesitan un sonido final y un silbido juntos.
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:103]:
```suggestion
  - las notas azules grandes necesitan un hitsound final y un silbido juntos.
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+- Los sliders representan la nota amarilla larga (también conocida como redoble).
+- El spinner representa un shaker.
+
+El diseño de la colocación de las notas no importa.
+
+### Notas de conversión de osu!
+
+Cuando se produce una conversión de mapa (jugando un mapa de osu! en modo osu!taiko), los sliders muy cortos (normalmente menos de un beat) se convierten automáticamente en notas rojas o azules, dependiendo del hitsound utilizado.
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:111]:
```suggestion
Cuando se produce una conversión de mapa (jugando un mapa de osu! en el modo osu!taiko), los sliders muy cortos (normalmente menos de un beat) se convierten automáticamente en notas rojas o azules, dependiendo del hitsound utilizado.
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+También ten en cuenta que los redobles de 1/6 se dan si se utiliza la velocidad de marca del slider de **3**.
+
+## Curiosidades
+
+### Jugabilidad
+
+- Jugar en un campo vacío no implica ningún fallo.
+- Redoble de tambores: El límite máximo de aciertos en el slider es:
+  - 4 veces la longitud del slider, o
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:125]:
```suggestion
  - 4 veces la longitud del slider, u
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+  - 8 veces la longitud del slider en canciones iguales o inferiores a 125 PPM.
+- A diferencia de los otros modos de juego, el *Kiai Time* tiene un efecto en las puntuaciones porque se refiere al *«Go-Go Time»* en *Taiko no Tatsujin*. Mientras *Kiai Time* está activo, el tambor de la parte superior izquierda cambia de animación (llamado *pippidon* o *Don*/*Katsu* en *Taiko no Tatsujin*), el campo de juego tiene un degradado de fondo y el área de golpe gana un gráfico de fuego a su alrededor.
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:127]:
```suggestion
- A diferencia de los otros modos de juego, el *Kiai Time* tiene un efecto en las puntuaciones porque se refiere al *«Go-Go Time»* en *Taiko no Tatsujin*. Mientras el *Kiai Time* está activo, el tambor de la parte superior izquierda cambia de animación (llamado *pippidon* o *Don*/*Katsu* en *Taiko no Tatsujin*), el campo de juego tiene un degradado de fondo y el área de golpe gana un gráfico de fuego a su alrededor.
```

File: wiki/Game_mode/osu!taiko/es.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+  - Además, todas las notas ganan un multiplicador de puntuación de 1,2x, incluidas las notas amarillas largas, excepto los golpes en un shaker (el golpe final sigue multiplicándose).
+- La mascota de osu!taiko es [pippidon](/wiki/Mascots#pippi).
+- Cuando se juega con [Auto](/wiki/Gameplay/Game_modifier/Auto), el nombre del jugador será *mekkadosu!*.
+
+### Historia
+
+![Logo de osu!taiko en modo especial](img/Taiko_logo.jpg "Logo de taiko en modo especial")
+
+- La opción `Utilizar la skin de Taiko para Taiko` en la barra lateral de opciones en la sección de skins utilizará los elementos de la carpeta `taiko` cuando juegue en *osu!taiko*, independientemente de los elementos taiko de la skin actual. La carpeta usada para mantener la skin *[Taiko por LuiginHann](https://osu.ppy.sh/community/forums/topics/41319)*, que se puede descargar desde el ahora desaprobado `osume.exe` (un archivo para actualizar osu! para cuando sea necesario actualizar) en la pestaña `Skin`.
```
Comment (UnknownOsu) [inline wiki/Game_mode/osu!taiko/es.md:136]:
```suggestion
- La opción `Utilizar la skin de Taiko para Taiko` en la barra lateral de opciones en la sección de skins utilizará los elementos de la carpeta `taiko` cuando juegues en *osu!taiko*, independientemente de los elementos taiko de la skin actual. La carpeta usada para mantener la skin *[Taiko por LuiginHann](https://osu.ppy.sh/community/forums/topics/41319)*, que se puede descargar desde el ahora obsoleto `osume.exe` (un archivo para actualizar osu! para cuando sea necesario actualizar) en la pestaña `Skin`.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Comment by UnknownOsu** [comment]
Tal vez también poner la foto en español en `controles` en `osu!catch`

**Review by UnknownOsu — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
> Tal vez también poner la foto en español en `controles` en `osu!catch`

for a needs_cleanup article it's probably fine.

**Pull request merged**